EIC-EA/EIC-community-D8
# EICNET-488: Implement content page template

### New features

- Installed EIC Community theme;
- Installed contrib module [Components!](https://www.drupal.org/project/components);
- Setup EIC Community components namespace to use within drupal twig templates;
- Create initial node page template;
- Create templates for node page paragraphs;

### Included paragraph templates

- Full text content
- Text and media content
- Full media content

### Missing paragraphs templates

- Banner
- Quote
- Tiles content

### Tests

- Create a new content page and insert the following paragraphs: Full text content, Text and media content, Full media content
- Check if the introduction header and the paragraphs are rendered according to the storybook.
